http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreterSpec.scala

```diff
@@ -734,71 +150,30 @@ class AKSInterpreterSpec extends AnyFlatSpecLike with TestComponent with Leonard
       } yield {
         app shouldBe defined
         app.get.app.status shouldBe AppStatus.Running
-        app.get.app.appAccessScope shouldBe Some(AppAccessScope.WorkspaceShared)
-        app.get.app.samResourceId.resourceType shouldBe SamResourceType.SharedApp
         app.get.cluster.asyncFields shouldBe defined
-        app
-      }
-
-      val dbApp = res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
-      dbApp shouldBe defined
-      val app = dbApp.get.app
-
-      val deletion = for {
-        _ <- aksInterp.deleteApp(DeleteAKSAppParams(app.appName, workspaceId, landingZoneResources, cloudContext))
-        deletedApp <- KubernetesServiceDbQueries
-          .getActiveFullAppByName(CloudContext.Azure(cloudContext), app.appName)
-          .transaction
-      } yield {
-        deletedApp shouldBe None
-        verify(mockAzureRelayService).deleteRelayHybridConnection(
-          RelayNamespace(ArgumentMatchers.eq(landingZoneResources.relayNamespace.value)),
-          RelayHybridConnectionName(ArgumentMatchers.eq(s"${app.appName.value}-${workspaceId.value}")),
-          ArgumentMatchers.eq(cloudContext)
-        )(any())
       }
 
-      deletion.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
+      res.unsafeRunSync()(cats.effect.unsafe.IORuntime.global)
     }
 
-  for (appType <- List(AppType.Wds, AppType.Cromwell, AppType.WorkflowsApp))
-    it should s"create ${appType} with wsm resources, then successfully delete them" in isolatedDbTest {
-      val mockAzureRelayService = setUpMockAzureRelayService
-
-      val aksInterp = new AKSInterpreter[IO](
-        config.copy(
-          wdsAppConfig = config.wdsAppConfig.copy(databaseEnabled = true),
-          coaAppConfig = config.coaAppConfig.copy(databaseEnabled = true),
-          workflowsAppConfig = config.workflowsAppConfig.copy(enabled = true)
-        ),
-        MockHelm,
-        mockAzureBatchService,
-        mockAzureContainerService,
-        mockAzureApplicationInsightsService,
-        mockAzureRelayService,
-        mockSamDAO,
-        mockCromwellDAO,
-        mockCbasDAO,
-        mockCbasUiDAO,
-        mockWdsDAO,
-        mockHailBatchDAO,
-        mockWsmDAO,
-        mockKube,
-        mockWsm
-      ) {
-        override private[util] def buildMsiManager(cloudContext: AzureCloudContext) = IO.pure(setUpMockMsiManager)
+  // update and poll each app type
+  // TODO
```

Comment:
OK this is a TODO -- I want to add some unit tests around upgrade (I didn't see any previously)